.gitignore

```diff
@@ -1,3 +1,4 @@
 *.gem
 .bundle
 pkg/*
+coverage
```

Comment:
I normally like to scope those to the git root and make clear that it's a folder: `/coverage/`